hedera-mirror-rest/viewmodel/blockViewModel.js

```diff
@@ -32,11 +32,11 @@ class BlockViewModel {
    * @param {Block} recordFile
    */
   constructor(recordFile) {
-    this.count = recordFile.count;
+    this.count = BigInt(recordFile.count);
     this.hapi_version = recordFile.getFullHapiVersion();
     this.hash = utils.addHexPrefix(recordFile.hash);
     this.name = recordFile.name;
-    this.number = recordFile.index;
+    this.number = BigInt(recordFile.index);
```

Comment:
The fields are already BigInt when returned by the database.
```suggestion
    this.number =recordFile.index;
```